storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -781,21 +884,19 @@ RS_Status WriteColToRespBuff(const NdbRecAttr *attr, PKRResponse *response) {
   }
   case NdbDictionary::Column::Bit: {
     //< Bit, length specifies no of bits
-    int32 attr_bytes = col->getSizeInBytes();
     Uint32 words     = attr->getColumn()->getLength() / 8;
     if (attr->getColumn()->getLength() % 8 != 0) {
       words += 1;
     }
 
     // change endieness
     int i = 0;
-    char reversed[words];
+    char reversed [MAX_TUPLE_SIZE_IN_WORDS*4];
```

Comment:
max size is 64 bits https://dev.mysql.com/doc/refman/8.0/en/bit-type.html
create a constant of size 8